reflectivity_ui/interfaces/data_manager.py

```diff
@@ -197,6 +197,9 @@ def add_active_to_reduction(self):
                     self.reduction_states = list(self.data_sets.keys())
                 is_inserted = False
                 q_min, _ = self._nexus_data.get_q_range()
+                if q_min is None:
```

Comment:
I have come across this problem too with `q_min` being `None`, but I haven't looked into why it happens. Do you know why it happens?

One issue is that in [load_data_from_reduced_file](https://github.com/neutrons/reflectivity_ui/blob/next/reflectivity_ui/interfaces/data_manager.py#L740) we don't check the return value from `add_active_to_reduction`, so I think the return statement here can make it fail to load a file silently. If `q_min` equals `None` is expected and normal, could you please add a check for the return value in `load_data_from_reduced_file` and a log message that loading the file failed?